x-pack/legacy/plugins/maps/public/inspector/views/register_views.ts

```diff
@@ -4,8 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { MapView } from './map_view';
+import { npSetup } from 'ui/new_platform';
 
-import { viewRegistry } from 'ui/inspector';
+// @ts-ignore
+import { MapView } from './map_view';
 
-viewRegistry.register(MapView);
+npSetup.plugins.inspector.registerView(MapView);
```

Comment:
I tried to move it to `plugin.ts` but there was an issue (an empty plugin page) when the uiExport was specified as `inspectorViews: ['plugins/maps/legacy']`. So I left view registration in a separate file.

Reply:
It's probably because that caused the maps plugin to be loaded twice. Definitely fine for the scope of this PR to leave it as is.